nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -73,11 +110,16 @@ def _apply_mask_and_get_affinity(seeds, niimg, radius, allow_overlap,
     mask_coords = mask_coords.astype(int).tolist()
     for i, seed in enumerate(seeds):
         try:
-            A[i, mask_coords.index(seed)] = True
+            A[i, mask_coords.index(list(map(int, seed)))] = True
```

Comment:
Sorry, why do we need that ?

Reply:
the `mask_coords` are converted to int above so unless the seed coords are too they will (almost) never be found in `mask_coords`